internal/adapters/storage/postgres/scheduler/db_scheduler.go

```diff
@@ -113,3 +116,31 @@ func (s *Scheduler) ToScheduler() (*entities.Scheduler, error) {
 		Autoscaling:     info.Autoscaling,
 	}, nil
 }
+
+func parseFromAnnotationEntityToMap(entities []*entities.Annotation) *map[string]string {
```

Comment:
now that annotation is a map, you don't really need this entity, you can just use the map instead

Reply:
it's ok to not declare an entity to annotations? If it's ok I going to change